egs/chime6/asr1/local/train_lms_srilm.sh

```diff
@@ -0,0 +1,261 @@
+#!/usr/bin/env bash
```

Comment:
We may not need this?

Reply:
This file is used in the data preparation too. They build LM for further use in the data cleanup.